gatewayconfig/bluetooth/services/helium_service.py

```diff
@@ -7,8 +7,8 @@
 from gatewayconfig.bluetooth.characteristics.mac_address_characteristic import MacAddressCharacteristic
 from gatewayconfig.bluetooth.characteristics.lights_characteristic import LightsCharacteristic
 from gatewayconfig.bluetooth.characteristics.wifi_ssid_characteristic import WifiSSIDCharacteristic
-from gatewayconfig.bluetooth.characteristics.assert_location_characteristic import AssertLocationCharacteristic
-from gatewayconfig.bluetooth.characteristics.add_gateway_characteristic import AddGatewayCharacteristic
+# from gatewayconfig.bluetooth.characteristics.assert_location_characteristic import AssertLocationCharacteristic
+# from gatewayconfig.bluetooth.characteristics.add_gateway_characteristic import AddGatewayCharacteristic
```

Comment:
the implementation for this is supposed to arrive only with alpha24 or higher of gateway-rs